modules/mgmt-creating-a-node-maintenance-cr.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/mgmt-node-maintenance.adoc
+
+[id="mgmt-creating-a-node-maintenance-cr_{context}"]
+= Setting a node to maintenance mode with a custom resource
```

Comment:
The id should be as close as approximation to the title as possible (shortcuts like vm or cli, or dropping articles is fine). The id should be identical to the filename (I heartily endorse copy/paste).

Reply:
refactored all the files and ids as you suggested. thanks.